src/views/GroupSettings.vue

```diff
@@ -28,6 +28,10 @@
 			{{ t('user_retention', 'Users are deleted when they did not log into their account within the given number of days. This will also delete all files of the affected users.') }}
 		</p>
 
+		<p v-if="ldapBackendEnabled" class="settings-hint">
+			{{ t('user_retention', 'Users from LDAP are deleted locally only, unless the LDAP write support app is enabled. When still available on LDAP, users will reappear.') }}
```

Comment:
```suggestion
			{{ t('user_retention', 'Users from LDAP are deleted locally only, unless the LDAP write support app is enabled. When still available on LDAP, users will reappear after a new login.') }}
```

Reply:
Nope, login is not required. A search op that'll yield the user is enough.